.travis.yml

```diff
@@ -12,7 +12,7 @@ addons:
     packages:
       - python2.4
 install:
-  - pip install tox PyYAML Jinja2 sphinx
+  - pip install tox PyYAML Jinja2 sphinx netaddr
```

Comment:
Thanks for the hint!
So by simply moving the netaddr test dependency from .travis.yml to the tox/requirements.txt everything should be fine?

Then I will try to make a new PR based on this.